crates/common/src/api/client.rs

```diff
@@ -184,6 +183,19 @@ impl std::fmt::Display for ApiErrorKind {
     }
 }
 
+impl crate::CliError for ApiError {
+    fn exitcode(&self) -> exitcode::ExitCode {
```

Comment:
Not needed in this PR, but can we remove the `exitcode` dependency? Can be vendored, it's just a set of constants.